src/ray/raylet/local_object_manager.h

```diff
@@ -365,6 +377,9 @@ class LocalObjectManager {
   /// The last time a restore log finished.
   int64_t last_restore_log_ns_ = 0;
 
+  /// The number of failed deletion requests.
+  std::atomic<int64_t> num_failed_deletion_requests_ = 0;
```

Comment:
Btw I think this doesn't have to be atomic (it is single threaded)

Reply:
hmm i thought the ` io_worker->rpc_client()->DeleteSpilledObjects` callback happens in a different thread.

Reply:
maybe we should start document explicitly where callbacks are run w.r.t threading model lol